projects/control-service/projects/base/src/main/java/com/vmware/taurus/base/FeatureFlags.java

```diff
@@ -26,4 +26,7 @@
    @Value("${featureflag.authorization.enabled:false}")
    boolean authorizationEnabled = false;
 
+   @Value("${datajobs.security.kerberos.enabled:false}")
+   boolean KRBAuthEnabled = false;
```

Comment:
variable names should start with lower-case letter
```suggestion
   boolean krbAuthEnabled = false;
```